index.html

```diff
@@ -5,9 +5,9 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <title>BC Business Registry</title>
-    <link rel="icon" href="<%= BASE_URL %>favicon.png">
-    <link rel="stylesheet" href="<%= BASE_URL %>css/addresscomplete-2.30.min.css">
-    <script type="text/javascript" src="<%= BASE_URL %>js/addresscomplete-2.30.min.js" defer></script>
+    <link rel="icon" href="/favicon.ico">
```

Comment:
This is the Vue icon, and should be the BC Gov icon instead. See if you can convert the PNG to an ICO. Or ask the UX folks where the BC Gov assets are and copy it from there.

There's this one but it doesn't have a transparent background:
https://github.com/bcgov/sbc-auth/blob/main/auth-web/public/favicon.ico

Reply:
This should be fixed now. Thanks for catching that Sev 😄

Reply:
Looks good.

![image](https://github.com/bcgov/business-create-ui/assets/10002889/dc9886eb-f18d-4f06-aae4-4b84838ea2db)

Travis, maybe you want to grab favicon.png from here to fix Auth Web? (There is an old ticket for this...)